controllers/reconcilers/grafana/ingress_reconciler.go

```diff
@@ -72,13 +72,13 @@ func (r *IngressReconciler) reconcileIngress(ctx context.Context, cr *v1beta1.Gr
 			return v1beta1.OperatorStageResultFailed, fmt.Errorf("ingress spec is incomplete")
 		}
 
-		status.AdminUrl = adminURL
+		cr.Status.AdminUrl = adminURL
 	}
 
 	return v1beta1.OperatorStageResultSuccess, nil
 }
 
-func (r *IngressReconciler) reconcileRoute(ctx context.Context, cr *v1beta1.Grafana, status *v1beta1.GrafanaStatus, _ *v1beta1.OperatorReconcileVars, scheme *runtime.Scheme) (v1beta1.OperatorStageStatus, error) {
+func (r *IngressReconciler) reconcileRoute(ctx context.Context, cr *v1beta1.Grafana, scheme *runtime.Scheme) (v1beta1.OperatorStageStatus, error) {
```

Comment:
I see that, unlike with other reconcilers, here you completely removed vars. Any reasons for that?

Reply:
Both were "Sub" reconcilers of the main `Reconcile` funciton, and neither used the `OperatorGrafanaReconciler` interface directly.
I've reverted it in favour of keeping it consistent across all the other Reconcile functions.